src/RequestData.java

```diff
@@ -47,6 +47,15 @@ public int getCurrentFloor() {
 		return this.currentFloor;
 	}
 	
+	/**
+	 * set the current floor number 
+	 * to the int currentFloor
+	 * @param int - currentFloor
+	 */
+	public void setCurrentFloor(int currentFloor) {
```

Comment:
Altering a request to reflect the current position of the elevator is tight coupling